src/components/modals/modalWrapper.tsx

```diff
@@ -27,14 +35,18 @@ const ModalWrapper: React.FC<ModalWrapperProps> = ({ children, modalKey }) => {
         />
       </Dialog.Overlay>
       {/* aria-describedby={undefined} is used to prevent the modal from being announced as a dialog */}
-      <Dialog.Content asChild aria-describedby={undefined}>
+      <Dialog.Content
+        asChild
+        aria-describedby={undefined}
+        className="max-w-[95%]"
+      >
         <motion.div
           className="ModalOuterContainer fixed inset-0 z-20 flex items-center justify-center"
           onClick={hideModal}
         >
           <AnimatePresence>
             <motion.div
-              className="ModalInnerContents absolute rounded-[50px] border-[3px] border-solid border-jade bg-cream px-14 pb-12 pt-14 text-jade drop-shadow-jade"
+              className="ModalInnerContents absolute -mt-10 flex max-h-[90%] max-w-[90%] rounded-[50px] border-[3px] border-solid border-jade bg-cream px-14 pb-12 pt-14 text-jade drop-shadow-jade"
```

Comment:
modal contents don't render well at 320px. Make sure you always check your site at 320px as this is about the smallest phones go